hack/make-rules/test-integration.sh

```diff
@@ -62,6 +63,8 @@ runTests() {
   make -C "${KUBE_ROOT}" test \
       WHAT="$(kube::test::find_integration_test_dirs ${2-} | paste -sd' ' -)" \
       KUBE_GOFLAGS="${KUBE_GOFLAGS:-} -tags 'integration no-docker'" \
+      # send the short directive so that long running tests can just integration compile and move on.
```

Comment:
You should remove the line above, right? [otherwise the next line won't have any effect in my opinion]